ansible/roles/util-cfg-service/templates/ui/de.properties.j2

```diff
@@ -130,6 +130,7 @@ org.iplantc.discoveryenvironment.workspace.defaultAppCategories      = {{ apps.u
 org.iplantc.discoveryenvironment.workspace.defaultBetaAppCategoryId  = {{ apps.beta_category }}
 org.iplantc.discoveryenvironment.workspace.defaultHpcAppCategoryId   = {{ apps.hpc_category }}
 org.iplantc.discoveryenvironment.workspace.defaultTrashAppCategoryId = {{ apps.trash_category }}
+org.iplantc.discoveryenvironment.workspace.ontologyAttrs             = {{ apps.ontology_attrs }}
```

Comment:
Since this value is only needed in this properties file, we don't need to create a group var for it.
I would just put the literal value here.